.github/workflows/stdlib_build_test_on_ballerina_lang_PRs.yml

```diff
@@ -0,0 +1,31 @@
+name: Standard library build test for ballerina-lang PRs
+
+on:
+  workflow_dispatch:
+      inputs:
+        ballerina_lang_branch:
+          description: 'Ballerina-lang branch'
+          required: true
```

Comment:
shall we make this optional and make `master` as the default branch?

Reply:
Sure I will add master as default.